android/build.gradle

```diff
@@ -24,7 +24,7 @@ dependencies {
 
     // api is used instead of implementation so the parent :app project can access any of the OneSignal Java
     //   classes if needed. Such as com.onesignal.NotificationExtenderService
-    api 'com.onesignal:OneSignal:3.15.3'
+    api 'com.onesignal:OneSignal:[4.0.0-beta2, 4.99.99]'
```

Comment:
Let's use specifically `4.0.0-beta2` so we know if someone is using exactly `react-native-onesignal-4.0.0-beta1` they have this.